src/mm/da/DataAccess.java

```diff
@@ -1377,4 +1377,22 @@ public boolean editMeetingNote(int meetingId, String message)
 		return true;
 	}
 
+	@Override
+	public String getAcadimicInsById(int id) {

```

Comment:
so all you want is the string? not the full object?

Reply:
now we need just the name to add to the constructor, if you want to send full object It is also possible.

Reply:
I would much prefer to send a full object whenever asked to access the db, in the future we will have to send all information and we can save work by sending it all. If you want to continue work you can complete this merge, I will make the required changes to support this choice for now.